From 9857e438437c88deb2d49b035bb4e1c5a697c398 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Tue, 20 Oct 2020 13:49:23 +0300
Subject: [PATCH 1233/1239] configs/mvebu: Add support for ASIX AX88179 USB2ETH

Add support for ASIX AX88179 USB2ETH adapters for Marvell boards
The adapter is detected as ax88179_eth device.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: Ic3f8026599d9e5cf2f07c65fd11339ccb339013c
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/boot/u-boot/+/38263
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Ofer Heifetz <oferh@marvell.com>
---
 configs/mvebu_comexpress-88f7020_defconfig  | 3 ++-
 configs/mvebu_crb_cn9130_defconfig          | 3 ++-
 configs/mvebu_crb_ep_cn9130_defconfig       | 3 ++-
 configs/mvebu_db-88f3720_defconfig          | 3 ++-
 configs/mvebu_db_armada8k_defconfig         | 3 ++-
 configs/mvebu_db_cn91xx_defconfig           | 3 ++-
 configs/mvebu_db_ep_cn91xx_defconfig        | 3 ++-
 configs/mvebu_espressobin-88f3720_defconfig | 3 ++-
 configs/mvebu_mcbin-88f8040_defconfig       | 3 ++-
 configs/mvebu_ocp-88f8040_defconfig         | 3 ++-
 configs/mvebu_ucpe-88f8040_defconfig        | 3 ++-
 11 files changed, 22 insertions(+), 11 deletions(-)

diff --git a/configs/mvebu_comexpress-88f7020_defconfig b/configs/mvebu_comexpress-88f7020_defconfig
index be7c609a0e..1badf00d6d 100644
--- a/configs/mvebu_comexpress-88f7020_defconfig
+++ b/configs/mvebu_comexpress-88f7020_defconfig
@@ -94,6 +94,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -102,4 +103,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
diff --git a/configs/mvebu_crb_cn9130_defconfig b/configs/mvebu_crb_cn9130_defconfig
index b699ca0024..9bf104f1cf 100644
--- a/configs/mvebu_crb_cn9130_defconfig
+++ b/configs/mvebu_crb_cn9130_defconfig
@@ -99,6 +99,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -107,4 +108,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
diff --git a/configs/mvebu_crb_ep_cn9130_defconfig b/configs/mvebu_crb_ep_cn9130_defconfig
index b18b96d38a..838c0eab29 100644
--- a/configs/mvebu_crb_ep_cn9130_defconfig
+++ b/configs/mvebu_crb_ep_cn9130_defconfig
@@ -102,6 +102,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -110,4 +111,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
diff --git a/configs/mvebu_db-88f3720_defconfig b/configs/mvebu_db-88f3720_defconfig
index f8d6afb0ec..dc8662cf47 100644
--- a/configs/mvebu_db-88f3720_defconfig
+++ b/configs/mvebu_db-88f3720_defconfig
@@ -87,6 +87,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -95,4 +96,4 @@ CONFIG_SHA256=y
 CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
-CONFIG_UPDATE_TFTP=y
\ No newline at end of file
+CONFIG_UPDATE_TFTP=y
diff --git a/configs/mvebu_db_armada8k_defconfig b/configs/mvebu_db_armada8k_defconfig
index a09f5f1e1d..0052d6809f 100644
--- a/configs/mvebu_db_armada8k_defconfig
+++ b/configs/mvebu_db_armada8k_defconfig
@@ -105,6 +105,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -113,4 +114,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
diff --git a/configs/mvebu_db_cn91xx_defconfig b/configs/mvebu_db_cn91xx_defconfig
index cc314efe33..5b26354988 100644
--- a/configs/mvebu_db_cn91xx_defconfig
+++ b/configs/mvebu_db_cn91xx_defconfig
@@ -95,6 +95,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -103,4 +104,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
diff --git a/configs/mvebu_db_ep_cn91xx_defconfig b/configs/mvebu_db_ep_cn91xx_defconfig
index caf8cb6899..cc5e47823c 100644
--- a/configs/mvebu_db_ep_cn91xx_defconfig
+++ b/configs/mvebu_db_ep_cn91xx_defconfig
@@ -99,6 +99,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -107,4 +108,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
diff --git a/configs/mvebu_espressobin-88f3720_defconfig b/configs/mvebu_espressobin-88f3720_defconfig
index 3d5970affe..1d869d253f 100644
--- a/configs/mvebu_espressobin-88f3720_defconfig
+++ b/configs/mvebu_espressobin-88f3720_defconfig
@@ -86,6 +86,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -94,4 +95,4 @@ CONFIG_SHA256=y
 CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
-CONFIG_UPDATE_TFTP=y
\ No newline at end of file
+CONFIG_UPDATE_TFTP=y
diff --git a/configs/mvebu_mcbin-88f8040_defconfig b/configs/mvebu_mcbin-88f8040_defconfig
index 8b3d7b57ab..05da113f8e 100644
--- a/configs/mvebu_mcbin-88f8040_defconfig
+++ b/configs/mvebu_mcbin-88f8040_defconfig
@@ -95,6 +95,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -102,4 +103,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
diff --git a/configs/mvebu_ocp-88f8040_defconfig b/configs/mvebu_ocp-88f8040_defconfig
index b550ef1ad2..b68d2393ca 100644
--- a/configs/mvebu_ocp-88f8040_defconfig
+++ b/configs/mvebu_ocp-88f8040_defconfig
@@ -90,6 +90,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -98,4 +99,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
diff --git a/configs/mvebu_ucpe-88f8040_defconfig b/configs/mvebu_ucpe-88f8040_defconfig
index bd84651db8..a7058ec599 100644
--- a/configs/mvebu_ucpe-88f8040_defconfig
+++ b/configs/mvebu_ucpe-88f8040_defconfig
@@ -83,6 +83,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_USB_ETHER_MCS7830=y
 CONFIG_USB_ETHER_RTL8152=y
 CONFIG_USB_ETHER_SMSC95XX=y
@@ -91,4 +92,4 @@ CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_UPDATE_TFTP=y
-CONFIG_OF_BOARD_SETUP=y
\ No newline at end of file
+CONFIG_OF_BOARD_SETUP=y
-- 
2.29.0

